Web/src/components/room/PermissionsDialog.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Modal from 'components/common/Modal';
+
+interface PermissionsDialogType {
```

Comment:
should this be props?
```suggestion
interface PermissionsDialogProps {
```